features/repost-activity.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Reposting an object
+  As a user
+  I want to repost an object in my feed
+  So that I can share the content with my followers
```

Comment:
Maybe a slight twist on this:

```
In order to share content with my followers
As a user
I want to be repost a post in my feed
```

Reply:
Updated in 9faffb4b1a509c7027d792aba0b493008496e9c9